schema/HPTUNE_CONFIG.md

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
same comments as config.md. Also, wondering if we should make a `docs/` directory to house these extra markdown files.

Reply:
Ack. `docs/` makes more sense than `schema/`. Updated in new commit.